### PR TITLE
Add NISRA QA content changes

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?

This adds latest NI schemas content changes and translations described on [the trello card](https://trello.com/c/VoTDGwJJ).

### How to review

Run quick launch schema, refer to Google docs "QA CHANGES" tab.

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-qa-changes/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-qa-changes/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by petechd** [comment]
[NISRA QA changes (S)](https://trello.com/c/VoTDGwJJ/4176-nisra-qa-changes-s)

**Comment by petechd** [comment]
Content now approved by Laura, some changes added - check column G of google doc.

**Review by berroar — APPROVED** [review]

File: source/jsonnet/northern-ireland/individual/blocks/identity-and-health/childhood_religion.jsonnet [hunk 1]
```diff
@@ -63,11 +63,11 @@ local question(title, otherDescription) = {
   page_title: 'Childhood religion',
   question_variants: [
     {
-      question: question(nonProxyTitle, 'You can enter your religion on the next question'),
+      question: question(nonProxyTitle, 'You can enter the religion on the next question'),
       when: [rules.isNotProxy],
     },
     {
-      question: question(proxyTitle, 'You can enter their religion on the next question'),
+      question: question(proxyTitle, 'You can enter the religion on the next question'),
```
Comment (ajmaddaford) [inline source/jsonnet/northern-ireland/individual/blocks/identity-and-health/childhood_religion.jsonnet:70]: As there is now no difference between the proxy and non-proxy descriptions, the description no longer needs to be passed as an argument to `question`.

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/identity-and-health/childhood_religion.jsonnet]: Removed, thanks!👍

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**